_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -59,6 +59,85 @@ The "Turn off" and "Remap" requirements from [Success Criterion 2.1.4][sc2.1.4]
 
 ## Test Cases
 
+<details>
+<summary>
+These Javascript and Style files are used in several examples:
+</summary>
+
+File [`/test-assets/ffbc54/shortcut.js](/test-assets/ffbc54/shortcut.js):
+
+```javascript
+const defaultParams = {
+	target: 'target',
+	focusOnly: false,
+	shortcutKey: '',
+	ctrlKey: false,
+	disabled: false,
+}
+
+const shortcutDefinitions = new Array()
+
+function activateShortcuts() {
+	document.body.addEventListener('keydown', function(event) {
+		for (const settings of shortcutDefinitions) {
+			if (!settings.disabled) {
+				const target = document.getElementById(settings.target)
+
+				if (
+					event.key === settings.shortcutKey &&
+					(!settings.ctrlKey || event.getModifierState('Control')) &&
+					(!settings.focusOnly || document.activeElement === target)
+				) {
+					document.getElementById('list').innerHTML += '<li>' + target.value + '</li>'
+					target.value = ''
+					event.preventDefault()
+				}
```

Comment:
I'm debating if we should consider 2 or 3 spaces per tab, or some other way of reducing the amount of indentations, since this is getting kinda close to the JavaScript pyramid of doom. For this specific example you could do:
```javascript
if (!settings.disabled) {
    continue;
}
```
which would get rid of at least 1 level of indentation.

Reply:
Yes. That's unrelated to this PR (the [existing file](https://act-rules.github.io/test-assets/ffbc54/shortcut.js) already has the problem). It might be linked to our linter rules 🤔 Not sure…